woocommerce/storefront
# Single Product Pagination

To test:

1. Checkout this branch
2. Go to Single Product page
3. You should see:

<img width="1384" alt="screen shot 2018-03-14 at 17 59 07" src="https://user-images.githubusercontent.com/1177726/37421547-71387014-27b1-11e8-9789-c6d51d88bab3.png">

On hover:

<img width="574" alt="screen shot 2018-03-14 at 17 59 51" src="https://user-images.githubusercontent.com/1177726/37421576-8973c3b8-27b1-11e8-87f6-5d6be942eca3.png">

Closes #766.
